include/crm/common/xml_internal.h

```diff
@@ -142,4 +142,34 @@ do {
  */
 void pcmk__xml_serialize_fd_formatted(int fd, xmlNode *cur);
 
+enum pcmk__xml_artefact_ns {
+    pcmk__xml_artefact_ns_legacy_rng = 1,
+    pcmk__xml_artefact_ns_legacy_xslt,
+    pcmk__xml_artefact_ns_base_rng,
+    pcmk__xml_artefact_ns_base_xslt,
+};
```

Comment:
spelling: artifact

Reply:
this is not a typo, it's a fully valid spelling